DocSum/docker/ui/react/.env

```diff
@@ -0,0 +1 @@
+VITE_BACKEND_SERVICE_ENDPOINT=http://backend_address:8888/v1/docsum
```

Comment:
The variable names in the .env file are not consistent with the ones you are actually using.

Reply:
renamed it to VITE_DOC_SUM_URL